packages/admin-ui/src/columns/columns.style.ts

```diff
@@ -0,0 +1,39 @@
+import type { CSSPropAutocomplete, SpaceTokens } from '@vtex/admin-ui-core'
+import { style, styleVariants } from '@vtex/admin-ui-core'
+
+export const columnsBaseline = (space: CSSPropAutocomplete<SpaceTokens>) =>
+  style({
+    display: 'flex',
+    flexWrap: 'wrap',
+    '> *:not(:first-child)': {
+      paddingLeft: space,
```

Comment:
I think this paddingLeft is causing the following:


![Screen Shot 2022-07-11 at 15 26 27](https://user-images.githubusercontent.com/991309/178332652-083dedc6-1770-4b00-a507-fd83694fd09d.png)

Reply:
it was because I was defining a default value of `12` to the units property